src/zrc/api/viewsets.py

```diff
@@ -256,8 +267,17 @@ def perform_create(self, serializer):
 
         super().perform_create(serializer)
 
+    def get_kenmerken(self, data):
+        kenmerken = get_kenmerken_from_model(
+            url=data['zaak'],
+            model=Zaak,
+            topics=settings.NOTIFICATIES_KENMERKEN_NAMES
+        )
+        return kenmerken
+
 
-class ZaakObjectViewSet(mixins.CreateModelMixin,
+class ZaakObjectViewSet(NotificationCreateMixin,
```

Comment:
I think using the ViewSetMixin is the better choice over CreateMixin, to prevent future errors.

Reply:
In this case I will add update() and destroy() actions to viewsets which don't support this actions.